spec/01-unit/03-conf_loader_spec.lua

```diff
@@ -1679,18 +1679,27 @@ describe("Configuration loader", function()
   end)
 
   describe("#wasm properties", function()
+    local tmp, cleanup
```

Comment:
```suggestion
    local tmp_path, cleanup
```
My pet peeve is vars named `tmp` 😅